### PR TITLE
ola: fix build on macos

### PR DESCRIPTION
**Pull request by svlobanov**
Latest Apple clang (v13) defines __cplusplus=199711 by default, but
protobuf requires at least 201103 (c++11)

This patch specifies -std=c++11 for host compiler to make it
compatible with protobuf requirements

Signed-off-by: Sergey V. Lobanov <sergey@lobanov.in>

Maintainer: @bk138 
Compile tested: (armvirt/64, OpenWrt trunk)

Related PR: https://github.com/openwrt/openwrt/pull/4964

Description: see above

### CODE REVIEW

**Comment by svlobanov** [comment]
```
sergey@Sergeys-MacBook-Air ola % g++ --version
Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 13.0.0 (clang-1300.0.29.30)
Target: arm64-apple-darwin20.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
sergey@Sergeys-MacBook-Air ola % cat 1.cpp
#include <iostream>

int main() {
    std::cout << __cplusplus ;
    return 0;
}                                                                                                                                           
sergey@Sergeys-MacBook-Air ola % g++ 1.cpp
sergey@Sergeys-MacBook-Air ola % ./a.out
199711                                                                                                                                
sergey@Sergeys-MacBook-Air ola % g++ -std=c++11 1.cpp
sergey@Sergeys-MacBook-Air ola % ./a.out
201103       
```

File: net/ola/Makefile [hunk 1]
```diff
@@ -75,7 +77,7 @@ endef
 
 define Host/Install
 	$(INSTALL_DIR) $(STAGING_DIR_HOSTPKG)/bin
-	$(CP) $(HOST_BUILD_DIR)/protoc/ola_protoc_plugin $(STAGING_DIR_HOSTPKG)/bin
+	$(INSTALL_BIN) $(HOST_BUILD_DIR)/protoc/ola_protoc_plugin $(STAGING_DIR_HOSTPKG)/bin
```
Comment (bk138) [inline net/ola/Makefile:80]: This is not really related, isn't it? A separate commit would've been better, but ok with me.

File: net/ola/Makefile [hunk 2]
```diff
@@ -64,6 +64,8 @@ HOST_CONFIGURE_ARGS += \
 	--disable-doxygen-html \
 	--disable-doxygen-doc
 
+HOST_CXXFLAGS = -std=c++11
```
Comment (bk138) [inline net/ola/Makefile:67]: x86-64 build test fails, you might want to have a look.

Comment (svlobanov) [reply net/ola/Makefile:67]: x86_64 build fail is a CI issue, it is not related to this commit. Almost all PRs fail on x86_64 build due to CI cryptodev issue. you may check another (merged and unmerged) PRs - the same story

File: net/ola/Makefile (hunk 1 above)
Comment (svlobanov) [reply net/ola/Makefile:80]: It is not related to macos build issue. If it is required I can split this commit to two commits

File: net/ola/Makefile (hunk 2 above)
Comment (bk138) [reply net/ola/Makefile:67]: OK!

**Comment by svlobanov** [comment]
https://github.com/openwrt/openwrt/pull/4964 applied, so this PR is ready to be merged

**Comment by neheb** [comment]
something bothers me here. configure.ac has a bunch of logic to set -std=c++11 and it seems to be failing.

**Comment by neheb** [comment]
oh maybe this needs to be backported: https://github.com/OpenLightingProject/ola/commit/30fe936a885c8a364bdda507a692c8e259576621

**Comment by svlobanov** [comment]
> oh maybe this needs to be backported: [OpenLightingProject/ola@30fe936](https://github.com/OpenLightingProject/ola/commit/30fe936a885c8a364bdda507a692c8e259576621)

@neheb done
